eppo_client/client.py

```diff
@@ -219,6 +221,100 @@ def get_assignment_detail(
             logger.error("[Eppo SDK] Error logging assignment event: " + str(e))
         return result
 
+    def get_bandit_action(
+        self,
+        flag_key: str,
+        subject_key: str,
+        subject_attributes: Attributes,
+        actions_with_contexts: List[ActionContext],
+        default: str,
+    ) -> BanditResult:
```

Comment:
Main function here, please make sure you like the choices I have made here:

- adding a default argument (think of this as variation, not action!)
- returning a BanditResult object that contains both variation and action (to cleanly separate whether the bandit has taken an action, or the choice is up to user)

Reply:
I think this is fine to try, and see what customers think.

As Mike Tyson says, "Everybody has a plan until they get punched in the face"

Reply:
Added the `to_string` method so people can have both options